### ISSUE TITLE
Translation : Telugu

### ISSUE THREAD

**Issue by aravindputrevu** [jwasham/google-interview-university#117]
I would like to translate to telugu

**Comment by jwasham** [comment]
Hi @aravindputrevu 
Thanks!
Create a fork and make a README-te.md file in the translations directory.

**Comment by mickeymanik** [comment]
When is it coming in telugu . eagerly waiting..

**Comment by aravindputrevu** [comment]
I started the work, but @mickeymanik lot of character are coming in a non-readable mode. I would soon resolve and complete it.

**Comment by dileep8014** [comment]
@aravindputrevu - any idea when you will be able to finish it , waiting to see it in action

**Comment by aravindputrevu** [comment]
@dileep8014  Thank you. I have started and got busy after some time. You can help me to convert some part of it!

**Comment by dileep8014** [comment]
@aravindputrevu  - sure let e know

**Comment by bharath-koteswarao** [comment]
Wow I never thought that I would see some thing in telugu in github.

Guys but what's the point in translating to telugu ? Can't someone who came till here don't understand english ? Just my opinion 👍

**Comment by vk927** [comment]
I agree with @bharath-koteswarao  no need to translate to Telugu. In industry we need to code in English and explain our code or algorithms to colleagues in English. Instead of that you guys can add more questions and concepts such as Convex Hull etc. Anyways I appreciate your effort.

**Comment by NaveenBen** [comment]
i am ready to put my hand in this telugu translation
